src/routes/collectibles.rs

```diff
@@ -158,18 +157,12 @@ pub async fn get_collectibles(
     trusted: Option<bool>,
     exclude_spam: Option<bool>,
 ) -> ApiResult<content::Json<String>> {
-    let url = format!(
-        "{}/v1/safes/{}/collectibles/?trusted={}&exclude_spam={}",
-        base_transaction_service_url(),
-        safe_address,
-        trusted.unwrap_or(false),
-        exclude_spam.unwrap_or(true)
-    );
-
-    Ok(content::Json(
-        RequestCached::new(url)
-            .request_timeout(collectibles_request_timeout())
-            .execute(context.client(), context.cache())
-            .await?,
-    ))
+    collectibles(
```

Comment:
I've extracted this into a method to match balances. We are not using `CacheResponse` here, not even prior to this change. We only cache the backend request. Is this ok? I would say so, as we don't map anything.